launch/shell.go

```diff
@@ -45,44 +44,8 @@ func (l *Launcher) launchWithShell(self string, process Process) error {
 }
 
 func (l *Launcher) profiles(process Process) ([]string, error) {
```

Comment:
What do you think about renaming this function to `getProfileDFiles` (or something similar starting with `get`)?

Reply:
I went with `getProfiles` because the returned values include `<app-dir>/.profile` if it exists which isn't in a `profile.d` directory.